README.md

```diff
@@ -3,8 +3,20 @@
 This project contains C# AltTester tests for a project using the New Input System.
 The tested actions are: tap object, key down/ key up, press key, click object, scroll, begin/end touch and tap simulated with swipe.
 
+## Before running the tests
+To run the tests, you must include the AltTester Unity SDK in the project. To do that, you can choose between the following ways:
+1. Add the AltTester Unity SDK submodule to the project
+    - in the <i> New Input System proj/Assets </i> folder add the submodule using the <i><b> git submodule add https://github.com/alttester/AltTester-Unity-SDK.git </b></i> command;
+    - make sure that the submodule added is on the master branch (you can use the following command <i><b> cd AltTester-Unity-SDK && git checkout master</b></i>).
```

Comment:
When a release is made for AltTester the user has to make a git pull on the master branch, maybe this info is useful too if they already have the project and thus are aware that the AltTester version was upgraded